$OpenBSD: patch-config-scripts_cups-compiler_m4,v 1.11 2019/08/25 09:59:18 ajacoutot Exp $

/usr/bin/ld: error: -shared and -pie may not be used together

Index: config-scripts/cups-compiler.m4
--- config-scripts/cups-compiler.m4.orig
+++ config-scripts/cups-compiler.m4
@@ -157,6 +157,8 @@ if test -n "$GCC"; then
 					AC_MSG_RESULT(no))
 				;;
 
+			openbsd*)
+				;;
 			*)
 				CFLAGS="$CFLAGS -fPIE -pie"
 				AC_TRY_COMPILE(,,[
